boardwalk/dev.yml

```diff
@@ -1,130 +0,0 @@
-version: '2'
```

Comment:
Why are all these lines being deleted?

Reply:
oops, that's unintentional, same for below. I'll correct it.